.github/ISSUE_TEMPLATE/ixd_user_story.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🧑‍🎨 IxD User story
+about: (DEV TEAM ONLY) A small chunk of work to be done by an Interaction Designer
+title: '(Fully descriptive title)'
+labels: 'IxD,triage'
+---
+
+<!-- Ensure the title can be understood without the parent item's context, e.g. "nimble-datepicker Interaction Design" rather than just "Interaction Design" -->
+
+## 📌 Request Background
+
+<!-- A short description of the request and requester. E.g. Who is the client and how do they expect to use the component? -->
+
+## Core Requirements
+
+<!-- Outline the essential requirements of the component. E.g. For a date picker… 
+
+- User must to be able to specify a _specific date_ for use in filter query
+- User should not be able to specify an invalid date
+- User must be able to enter date with keyboard interaction alone
+- … -->
+
+## Prior Art
+
+<!-- Provide an overview of other common implementations of this component. What does FAST provide out of the box? Material, AzDO, [others](https://storybook.js.org/showcase)?
+
+| Design System | Positives | Negatives |
+|---------------|-----------|-----------|
+| FAST          | …         | …         |
+| Material      | …         | …         | -->
+
+## Acceptance Criteria
+
+- [ ] IxD Spec submitted (link)
+- [ ] Visual Design work item created with latest requirements
```

Comment:
Should we add a bullet here describing the handoff to dev? e.g. "IxD Spec presented to feature lead or dev team"

Reply:
I kept it generic `IxD Spec reviewed by dev team`, but let me know if you'd like to be explicit about saying it should be presented to the team in a team meeting, in addition to reviewing and approving the IxD spec.